src/engine/Engine.cpp

```diff
@@ -145,6 +145,10 @@ void Engine::exportInputQueryWithError( String errorMessage )
 
 bool Engine::solve( unsigned timeoutInSeconds )
 {
+    // TODO: Remove this block after getting ready to support sigmoid with MILP.
+    if ( _preprocessedQuery.getTranscendentalConstraints().size() > 0 )
+        throw MarabouError( MarabouError::FEATURE_NOT_YET_SUPPORTED, "Marabou doesn't support sigmoid for solve yet." );
```

Comment:
Throw this error also in InputQuery::saveQuery()

Reply:
Added.